report_xlsx/report/report_xlsx.py

```diff
@@ -17,8 +17,37 @@
 class ReportXlsxAbstract(models.AbstractModel):
     _name = 'report.report_xlsx.abstract'
 
+    def _get_objs_for_report(self, docids, data, ids=None):
```

Comment:
Why the extra `ids` arg?

Reply:
The only reason was to allow for a clean inheritance chain if a wizard in particular wanted to use something other than data["context"]["active_ids"] to determine the active records for a report, it could pass ids and skip the logic in this functions.

I have no personal use for it.

Reply:
@pedrobaeza I removed it, you are right, not really needed.